src/components/containers/Sidebar/Sidebar.js

```diff
@@ -1,12 +1,31 @@
 import React, { Component } from 'react';
 
 import './Sidebar.css';
+import github from '../../../images/github.png';
+import info from '../../../images/blueinfo.png';
+import youtube from '../../../images/youtube.png';
+import bluedocumentation from '../../../images/bluedocumentation.png';
+import purple from '../../../images/Purple.png';
```

Comment:
My heart tells me there is a prettier way to do these imports, but I'm inclined to leave it be, since it doesn't affect functionality, and I don't remember if there's any extra config stuff we have to do to make it look nice.

Reply:
¯\_(ツ)_/¯ kk!